notebook/static/notebook/js/searchandreplace.js

```diff
@@ -147,58 +147,54 @@ define(function(require){
    * Search N' Replace action handler.
    **/
   var snr = function(env, event) {
-    var search  = $("<input/>")
-      .addClass('form-control')
-      .attr('placeholder','Find');
+      
     var isRegExpButton = $('<button/>')
       .attr('type', 'button')
       .attr('id', 'isreg')
-      .addClass("btn btn-default")
+      .addClass("btn btn-default btn-sm")
       .attr('data-toggle','button')
+      .css('font-weight', 'bold')
       .attr('title', 'Use regex (JavaScript regex syntax)')
       .text('.*');
 
-    var onlySelectedButton = $('<button/>')
+    var onlySelectedButton = $('<button><i class="fa fa-align-left"/></button>')
```

Comment:
doing that with jquery is bad. `$()` is not ment for templating, and it's inconsistent will al the rest of our code, like we won't be able to  for an `grep addClass | grep fa-align`